### DISCUSSION TITLE
Hide titlebar for MacOS (no traffic lights, without transparency)

### DISCUSSION THREAD

**Discussion by harrismcc** [tauri-apps/tauri#7587] [Q&A]
I've got a window for my app that appears from the toolbar, and so it doesn't need the titlebar. (Closed by clicking outside the window, can't be moved)

I don't want to turn off decorations and implement the rounded corners myself with transparency & css, as that would require making my app ineligible for the app store. However, it doesn't seem like there is a way to do this with "titleBarStyle"

Is there a workaround? Maybe I have to tap into cocoa and NSWindow directly?

**Comment by FabianLars** [comment]
Does the `hiddenTitle` window config does what you're asking for? Can't remember what exactly it does, probably only removes the text, not the buttons. Anyway, if it's not enough, please feel free to open a feature request for this :)
